pyproject.toml

```diff
@@ -3,3 +3,4 @@ line-length = 120
 
 [tool.isort]
 line_length=120
+known_first_party=['b', 'utility']
```

Comment:
Should we also include `dockerise`?

Reply:
I didn't bother because it no longer lives there as of #376